.github/workflows/build_and_run_idf_examples.yml

```
@@ -0,0 +1,86 @@
+name: ESP-IDF USB Device examples
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        idf_ver: ["release-v5.1", "release-v5.2", "release-v5.3", "release-v5.4", "release-v5.5", "latest"]
+      fail-fast: false
+    runs-on: ubuntu-latest
+    container: espressif/idf:${{ matrix.idf_ver }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Python dependencies
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          pip install --no-cache-dir idf-component-manager>=2.1.2 idf-build-apps==2.13.1 pyyaml --upgrade
+      - name: Build Network USB related examples
+        shell: bash
+        run: |
+          . ${IDF_PATH}/export.sh
+          export EXAMPLES_PATH="${IDF_PATH}/examples/network/sta2eth"
+          export MANIFEST_FILE="${IDF_PATH}/examples/network/.build-test-rules.yml"
```

Seems that I forgot to add the one important step for this verification in https://github.com/espressif/tinyusb/pull/58

```suggestion
          export MANIFEST_FILE="${IDF_PATH}/examples/network/.build-test-rules.yml"
          python .github/ci/override_managed_component.py tinyusb . ${EXAMPLES_PATH}
```